lms/djangoapps/instructor_task/api.py

```diff
@@ -337,7 +337,7 @@ def submit_calculate_students_features_csv(request, course_key, features):
     """
     task_type = 'profile_info_csv'
     task_class = calculate_students_features_csv
-    task_input = {'features': features}
+    task_input = features
```

Comment:
I'm also nervous about this fix long-term. It doesn't seem to address the underlying issue.

@adampalay I believe the old fix didn't work because the workers didn't have the correct microsite configuration for the task.